app/controllers/admin/users_controller.rb

```diff
@@ -3,7 +3,8 @@ class Admin::UsersController < Admin::AdminController
   def show
     @user = current_user
     @orders = current_user.orders
-    @all_orders = Order.all.order(created_at: :desc)
+    @all_orders = Order.all.order(created_at: :desc).
+    paginate(:page => params[:page], :per_page => 40)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Align the operands of an expression in an assignment spanning multiple lines.